### PR TITLE
Process Action: Assistant Builder support for schema

### PR DESCRIPTION
**Pull request by spolu**
## Description

- Generate is hard coded for now (will tie it to an endpoint in follow-up PR)
- Allows creation and edition of a schema

This is quite cargo-culted on our work on Extract

Screenshot from 2024-04-30 15-15-29
Screenshot from 2024-04-30 15-14-33

(not completely broken on mobile)

## Risk

- N/A (all beta)

## Deploy Plan

- deploy `front`

### CODE REVIEW

**Review by lasryaric — CHANGES_REQUESTED** [review]
Left 2 questions that I think require some changes. LMK.

File: front/components/assistant_builder/actions/ProcessAction.tsx [hunk 1]
```diff
@@ -104,10 +339,7 @@ export function ActionProcess({
         1000 pages book). Learn more about this feature in the{" "}
         <Hoverable
           onClick={() => {
-            window.open(
-              "https://dust-tt.notion.site/Table-queries-on-Dust-2f8c6ea53518464b8b7780d55ac7057d",
-              "_blank"
-            );
+            window.open("https://foo", "_blank");
```
[the remark this answers is not in the document]
Comment (spolu) [reply front/components/assistant_builder/actions/ProcessAction.tsx:342]: The url does not exist yet

File: front/components/assistant_builder/actions/ProcessAction.tsx [hunk 2]
```diff
@@ -10,12 +23,234 @@ import type { AssistantBuilderState } from "@app/components/assistant_builder/ty
...
+          <label
+            htmlFor={name}
+            className="block text-sm font-medium text-gray-700"
+          >
+            {label}
+          </label>
+        </div>
+      )}
+      <div className="mt-1 flex rounded-md shadow-sm">
+        <input
```
[the remark this answers is not in the document]
Comment (spolu) [reply front/components/assistant_builder/actions/ProcessAction.tsx:82]: Cargo culted from Extract. Will look into using sparkle Input. The look and feel might be a bit heavy with our sparkle Input IMHO but will double check.

**Comment by spolu** [comment]
cc @Duncid I'm misusing a bit the labels of Input to create column labels :grimacing: Also DropdownMenu does not have a label which could be nice to add since I hacked it here :/

File: front/components/assistant_builder/actions/ProcessAction.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (spolu) [reply front/components/assistant_builder/actions/ProcessAction.tsx:301]: (this is all behind flag)

**Review by lasryaric — APPROVED** [review]
LGTM

**Pull request merged**